tests/test_integration.py

```diff
@@ -183,20 +182,12 @@ def test_robot_default_name_arg(self):
 
     def test_robot_verbose(self):
         ''' If no suite name is configured, all suites must be taken into account '''
-        with patch('sys.stdout', new=StringIO()) as fake_out:
+        with self.assertLogs(level="INFO") as fake_out:
             retval = warnings_wrapper(['--verbose', '--robot', '--name', 'Suite Two', 'tests/test_in/robot_double_fail.xml'])
-        stdout_log = fake_out.getvalue()
+        stdout_log = fake_out.output
 
         self.assertEqual(1, retval)
-        self.assertEqual(
-            '\n'.join([
-                "Suite One &amp; Suite Two.Suite Two.Another test",
-                "Suite 'Suite Two': 1 warnings found",
-                "Counted failures for test suite 'Suite Two'.",
-                "Number of warnings (1) is higher than the maximum limit (0). Returning error code 1.",
-            ]) + '\n',
-            stdout_log
-        )
+        self.assertIn("INFO:root:Suite One &amp; Suite Two.Suite Two.Another test", stdout_log)
```

Comment:
Is this all the logs that the end user will see when using the CLI?

Reply:
Resolved in cca6fc5793584fae62b19c6db2c757bdeb930238

Reply:
JUnit, Robot, Polyspace, Sphinx,...

Proper capitalization of the tool names in the logs would look nicer.

Reply:
Resolved in 6e4e05299753c742d4c85afbd6a5a52162945938